### PR TITLE
Fixed bug in Climatic Summary dialog

### LINKED ISSUE #7670: Minor bug in the Climatic Summary Dialogue
The Within Year receiver should fill automatically the month by default
image

### PR DESCRIPTION
**Pull request by Fidel365**
Fixes #7670 
@N-thony , this PR is ready for preview

### CODE REVIEW

**Comment by N-thony** [comment]
> Fixes #7670 @N-thony , this PR is ready for preview

@Fidel365 can you imporve the title of this PR more clear. When you say `minor change` it is not clear what this PR seeks to address.

File: instat/dlgClimaticSummary.vb [hunk 1]
```diff
@@ -87,7 +87,8 @@ Public Class dlgClimaticSummary
 
         ucrReceiverWithinYear.SetParameter(New RParameter("within_variable", 2, False))
         ucrReceiverWithinYear.SetParameterIsString()
-        'ucrReceiverWithinYear.strSelectorHeading = "Factors"
```
Comment (N-thony) [inline instat/dlgClimaticSummary.vb:90]: any reason of removing this?

**Review by N-thony — CHANGES_REQUESTED** [review]

**Comment by Fidel365** [comment]
@N-thony , It did not see it's relevance there

**Comment by N-thony** [comment]
> @N-thony , It did not see it's relevance there

it is needed to give a user an idea from the selector which type of variable is expected to the corresponding receiver. 
So, I suggest that you put it back.

**Comment by Fidel365** [comment]
@N-thony , this PR is ready for re-review

**Comment by EstherNjeriLiberatta** [comment]
> > Fixes #7670 @N-thony , this PR is ready for preview
> 
> @Fidel365 can you improve the title of this PR to make it more clear? When you say `minor change` it is not clear what this PR seeks to address.

@Fidel365 Please make your title clearer as addressed by @N-thony. Which dialogue is this and what minor changes have you made? It's important especially to avoid working on similar dialogs. Thank you.

**Comment by N-thony** [comment]
> @N-thony , this PR is ready for re-review

@Fidel365 did you resolve my comment above?

**Comment by Fidel365** [comment]
@N-thony , I have changed the title of this PR

**Comment by Fidel365** [comment]
@N-thony , can you review this?

**Review by N-thony — CHANGES_REQUESTED** [review]

**Review by rdstern — DISMISSED** [review]
@N-thony from my point of view this seems fine now.  If the code is ok, then perhaps @lloyddewit could merge this.

File: instat/dlgClimaticSummary.vb [hunk 2]
```diff
@@ -87,6 +87,8 @@ Public Class dlgClimaticSummary
 
         ucrReceiverWithinYear.SetParameter(New RParameter("within_variable", 2, False))
         ucrReceiverWithinYear.SetParameterIsString()
+        ucrReceiverWithinYear.SetClimaticType("month")
+        ucrReceiverWithinYear.bAutoFill = True
         'ucrReceiverWithinYear.strSelectorHeading = "Factors"
```
Comment (lloyddewit) [inline instat/dlgClimaticSummary.vb:92]:
```suggestion
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Pull request merged**